docs/cmake-presets.md

```diff
@@ -0,0 +1,272 @@
+
+The following settings will be ignored when `CMakePresets.json` integration is enabled. Ignored settings will be logged to the Output Window when you run **CMake: Configure**.
+
+| Ignored setting in settings.json | `CMakePresets.json` equivalent |
+|--|--|
+| 'cmake.buildArgs` | Various options in `buildPreset` |
+| 'cmake.buildDirectory` | `configurePresets.binaryDir` |
+| 'cmake.buildEnvironment` | `buildPresets.environment` |
+| 'cmake.buildToolsArgs` | `buildPresets.nativeToolOptions` |
+| 'cmake.cmakePath` | `configurePresets.cmakeExecutable` |
+| 'cmake.configureArgs` | Various options in a `configurePreset` |
+| 'cmake.configureEnvironment` | `configurePresets.environment` |
+| 'cmake.configureSettings` | `configurePresets.cacheVariables` |
+| 'cmake.ctestParallelJobs` | `testPresets.execution.jobs` |
+| 'cmake.ctestArgs` | Various options in a `testPreset` |
+| 'cmake.defaultVariants` | Doesn't apply |
+| 'cmake.environment` | `configurePresets.environment` |
+| 'cmake.generator` | `configurePresets.generator` |
+| 'cmake.ignoreKitEnv` | Doesn't apply |
+| 'cmake.installPrefix` | `configurePresets.cacheVariables.CMAKE_INSTALL_PREFIX` |
+| 'cmake.parallelJobs` | buildPresets.jobs |
+| 'cmake.platform` | 'configurePresets.architecture` |
+| 'cmake.preferredGenerators` | `configurePresets.generator` |
+| 'cmake.setBuildTypeOnMultiConfig` | 'configurePresets.cacheVariables.CMAKE_BUILD_TYPE` |
+| 'cmake.testEnvironment` | `testPresets.environment` |
+| `cmake.toolset` | `configurePresets.toolset` |
+
+## Unsupported commands
+
+The following commands are not supported when `CMakePresets.json` integration is enabled:
+* **CMake: Quick Start**
+* **CMake: Select Variant**
+* **CMake: Scan for Kits**
+* **CMake: Select a Kit**
+* **CMake: Edit User-Local CMake Kits**
+
+## Troubleshooting
+
+If things aren’t working as expected, there are a few troubleshooting steps that you can take.
+
+If either `CMakePresets.json` or `CMakeUserPresets.json` is invalid, then none of the presets in the invalid file will be available for selection. CMake Tools IntelliSense can help you catch many of these JSON errors, but it won’t know if you're referencing a preset with `inherits` or `configurePreset` by the wrong name. To check if your preset files are valid, run `cmake --list-presets` from the command line at the root of your project directory (CMake 3.20 or higher is required). If either file is invalid, then you'll see the following error:
+
+```DOS
+CMake Error: Could not read presets from
+C:/Users/<user>/source/repos/<project-name>: JSON parse error
+```
+
+If you're working on Windows then CMake must be on the `PATH`.
+
+Other troubleshooting steps include:
+* Confirm cmake.exe and your generator are installed and on the `PATH`.
```

Comment:
Revise so this reads: 
Confirm cmake.exe and your generator are installed and on the `PATH`. If you are working on Windows, run the **CMake: Scan for Compilers** command so the extension can detect build tools installed with Visual Studio.

Reply:
done